components/list/list-item-drag-drop-mixin.js

```diff
@@ -884,6 +913,17 @@ export const ListItemDragDropMixin = superclass => class extends superclass {
 				@touchcancel="${this._onTouchCancel}"
 				@mousedown="${this._onDragTargetMouseDown}"
 				>
+				${content ? html`
+					<div slot="outside-control" style="width: 1.5rem;"></div>
+					${this._hasColorSlot ? html`
+					<div slot="color-indicator" class="d2l-list-item-color-outer" style="width: var(--d2l-list-item-color-width, 6px);"></div>` : nothing}
+					<div slot="expand-collapse" class="d2l-list-expand-collapse"></div>
+					<div slot="control"></div>
+					<div slot="actions" class="d2l-list-item-actions-container">
+						<slot name="actions" class="d2l-list-item-actions"></slot>
```

Comment:
None of the other slotted divs actually do anything, they just get styled. This `actions` slot does actually get the actions put in it, allowing for proper spacing there.